pkg/kubernetes/cluster_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes_test
+
+import (
+	"bytes"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubernetes-sigs/minibroker/pkg/kubernetes"
+)
+
+var _ = Describe("Cluster", func() {
+	It("should fail when reading the resolv.conf reader fails", func() {
+		var resolvConf failReader
+		clusterDomain, err := kubernetes.ClusterDomain(&resolvConf)
+		Ω(clusterDomain).Should(BeZero())
```

Comment:
Please don't use Ω — it's hard to type on non-Mac machines and harder to read.

Reply:
I use openSUSE (I work for SUSE - even if I didn't I'd probably still be using openSUSE) and I could copy-paste it! Let it be cool!

Reply:
No, let's not: I'm aiming for readability & ease of modification.

Speaking as somebody who _could_ (and sometimes does) type in a non-Latin language… Identifiers should still be in English.  Comments are more of a free-for-all, though.

Reply:
Whatever. ffd8738